mobile/src/home-view.js

```diff
@@ -170,8 +170,8 @@ class HomeView extends PureComponent {
           return name === item.replace('-Comment', '')
         })
         if (question) {
-          const answer = JSON.stringify(origResults[item])
-          let questionTitle = question.title ? question.title : question.name
+          const answer = getDefaultLocale(origResults[item])
```

Comment:
What was the JSON.stringify for?  We don't need it anymore?

Reply:
I believe this is an old fix for matrix questions acting as objects and causes a crash when questions (keys) had periods ending them

Reply:
Good catch